playbook.yml

```diff
@@ -6,9 +6,14 @@
       selinux:
         policy: targeted
         state: permissive
-    - name: 'install python3'
-      raw: sudo dnf -y install python3
-    - name: 'install libselinux-python3'
-      raw: sudo dnf -y install libselinux-python3
+    - name: Ensure Python 3 is available
+      dnf:
+        name: python3
+        state: present
+    - name: Install libselinux-python3
+      dnf:
+        name: libselinux-python3
+        state: present
   roles:
     - ansible-keylime-tpm20
+    - ansible-keylime-user
```

Comment:
Just noticed this was added a no role is there with this name:

```
ERROR! the role 'ansible-keylime-user' was not found in /vagrant/roles:/home/vagrant/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/vagrant

The error appears to be in '/vagrant/playbook.yml': line 19, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    - ansible-keylime-tpm20
    - ansible-keylime-user
      ^ here
Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

Reply:
@axelsimon  ^

Reply:
Good catch, thanks! It's indeed a mistake and should not have been there. It's part of the role i was using to test the Keylime dedicated user.

I'll prepare a PR that removes that.

Reply:
Here it is: https://github.com/keylime/ansible-keylime-tpm-emulator/pull/51